### PR TITLE
Preview image doesn't change when select new avatar in profile page

### LINKED ISSUE #781: Submit new profile will not work on Chrome
## Describe the bug

If you create a new developer profile, then the main form submit will not work on Chrome, because 1) the avatar field is mandatory 2) the avatar field uses createObjectURL, which is not tolerated anymore for a media object on Chrome.

## To reproduce

0. Use Google Chrome
1. Register
2. Click on "I'm looking for work"
3. Fill the whole form, scroll down to 'Save', and click on 'Save'.

## Expected behavior

The profile is successfully created

## Actual behavior

Avatar field is in error. Chrome Console shows  : Failed to execute 'createObjectURL' on 'URL' 

See StackOverflow answer : https://stackoverflow.com/a/53821674/2595513
"This error is caused because the function createObjectURL no longer accepts media stream object for Google Chrome"

Please note that there are no bug when using Firefox.

## Screenshots

I don't have a screenshot, alas

## Additional context

No additional context

### PR DESCRIPTION
**Pull request by ryy**
rel #781

Preview image doesn't change when select new avatar in profile page

## Pull request checklist
- [ ] My code contains tests covering the code I modified
  - sorry, I couldn't work system spec.
- [ ] I linted and tested the project with `bin/check`
  - All passed except system spec
- [x] I added significant changes and product updates to the [changelog](CHANGELOG.md)

## Evidence
https://user-images.githubusercontent.com/13917558/221443418-83bd612a-d96b-497d-8068-d4d21e81acad.mov

### CODE REVIEW

File: app/javascript/controllers/file_upload_controller.js [hunk 1]
```diff
@@ -6,8 +6,17 @@ export default class extends Controller {
...
+    this.imageTarget.src = objectUrl
+    this.imageTarget.srcset = objectUrl
+    // NOTE: Update the srcset of the <source> tag of the <picture> tag sub-element
+    const picture = event.currentTarget.closest('div').querySelector('picture');
+    if(picture) {
+      const sources = picture.querySelectorAll('source')
+      sources.forEach(source => {
+        source.srcset = objectUrl
+      });
+    }
```
Comment (joemasilotti) [inline app/javascript/controllers/file_upload_controller.js:19]: Can we convert this to another Stimulus target? Iterating over `this.pictureTargets` feels a lot cleaner.

**Review by joemasilotti — CHANGES_REQUESTED** [review]

**Comment by joemasilotti** [comment]
Hey again @ryy, do you still have appetite to finish this PR? If not, no worries! Let me know and I can take it over the finish line if need be.

**Review by joemasilotti — APPROVED** [review]

**Comment by joemasilotti** [comment]
Thanks for your contribution @ryy!

**Comment by ryy** [comment]
@joemasilotti 
Sorry, I was a little busy.
Thank you for help me.

**Pull request merged**